.golangci.yml

```diff
@@ -37,6 +37,28 @@ linters-settings:
       const:
         YEAR: '2023'
     template-path: .copyright-header.tmpl
+  gomnd:
+    checks:
+      - argument
+      # - case
+      # - condition
```

Comment:
I think it's worth checking this too. I think it's where most magic numbers occurs and one of the hardest to know why we set that number in the if statement. WDYT?

Reply:
i've activated them and fixed the numbers, i have couple which i am not sure of the meaning, maybe you can give me some feedback?

```
cmd/up/activate.go:191:27: mnd: Magic number: 3, in <condition> detected (gomnd)
pkg/k8s/volumes/crud.go:195:14: mnd: Magic number: 5, in <condition> detected (gomnd)
pkg/registry/image.go:94:34: mnd: Magic number: 2, in <condition> detected (gomnd)
```

Reply:
`cmd/up/activate.go:191:27: mnd: Magic number: 3, in <condition> detected (gomnd)`: We run a start script on the pod and get three lines as the output for getting some information. Should we call it `minStartScriptLines`?
`pkg/k8s/volumes/crud.go:195:14: mnd: Magic number: 5, in <condition> detected (gomnd)`: This magic number is to print the log every 5 times so we don't get too much logs with the same information. Maybe we can call it `logFrequency`?
`pkg/registry/image.go:94:34: mnd: Magic number: 2, in <condition> detected (gomnd)`: This magic number is the one that checks that the image is composed as `registry.com/image`. Maybe we can call it `registryImageParts`?

WDYT?